src/app/admin/metric-downloads/metric-downloads.component.html

```diff
@@ -1,35 +1,41 @@
 <main id="content" class="bypass-block-target">
-    
-    <div class="col-md-10 no-padding">
-      <div class="report-section">
-        <h1 class="srt-blue analytic-big-title" id="anchor-scanned-sol-by-date-metrics">Scanned Solicitation by Date Metrics</h1>
-        <p>
-          The Solicitation Download Report is a CSV file with the counts of newly added or updated solicitations
-          by date.
-        <button id="downloadScannedSolicitationsCSV" class="srt-button" style="float: none;" (click)="downloadScannedSolicitationsCSV()" >Download CSV</button>
-        </p>
-      </div>
+  <section class="usa-section">
+    <div class="usa-grid">
+      <div class="usa-width-two-thirds">
+        
+        <!-- Scanned Solicitation by Date Metrics -->
+        <div class="usa-section">
+          <h1 class="usa-heading" id="anchor-scanned-sol-by-date-metrics">Scanned Solicitation by Date Metrics</h1>
+          <p>
+            The Solicitation Download Report is a CSV file with the counts of newly added or updated solicitations by date.
+          </p>
+          <button class="usa-button" id="downloadScannedSolicitationsCSV" (click)="downloadScannedSolicitationsCSV()">
```

Comment:
I think the float:none style isn’t needed here since the USWDS classes (usa-grid, usa-width-*) handle all the layout and alignment. These classes use Flexbox, so they’re already taking care of spacing and positioning

does that make sense?